janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -0,0 +1,190 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.diskstorage.cql;
+
+import org.janusgraph.diskstorage.configuration.ConfigElement;
+import org.janusgraph.diskstorage.configuration.ConfigNamespace;
+import org.janusgraph.diskstorage.configuration.ConfigOption;
+import org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration;
+import org.janusgraph.graphdb.configuration.PreInitializeConfigOptions;
+
+/**
+ * Configuration options for the CQL storage backend. These are managed under
+ * the 'cql' namespace in the configuration.
+ */
+@PreInitializeConfigOptions
+public interface CQLConfigOptions {
+
+    public static final ConfigNamespace CQL_NS = new ConfigNamespace(
+            GraphDatabaseConfiguration.STORAGE_NS,
+            "cql",
+            "CQL storage backend options");
+
+    public static final ConfigOption<String> KEYSPACE = new ConfigOption<>(
+            CQL_NS,
+            "keyspace",
+            "The name of JanusGraph's keyspace.  It will be created if it does not exist.",
+            ConfigOption.Type.LOCAL,
+            "janusgraph");
+
+    public static final ConfigOption<String> READ_CONSISTENCY = new ConfigOption<>(
+            CQL_NS,
+            "read-consistency-level",
+            "The consistency level of read operations against Cassandra",
+            ConfigOption.Type.MASKABLE,
+            CQLStoreManager.CONSISTENCY_QUORUM);
+
+    public static final ConfigOption<String> WRITE_CONSISTENCY = new ConfigOption<>(
+            CQL_NS,
+            "write-consistency-level",
+            "The consistency level of write operations against Cassandra",
+            ConfigOption.Type.MASKABLE,
+            CQLStoreManager.CONSISTENCY_QUORUM);
+
+    // The number of statements in a batch
+    public static final ConfigOption<Integer> BATCH_STATEMENT_SIZE = new ConfigOption<>(
+            CQL_NS,
+            "batch-statement-size",
+            "The number of statements in each batch",
+            ConfigOption.Type.MASKABLE,
+            20);
+
+    // Whether to use unlogged batches
+    public static final ConfigOption<Boolean> ATOMIC_BATCH_MUTATE = new ConfigOption<>(
+            CQL_NS,
+            "atomic-batch-mutate",
+            "True to use Cassandra atomic batch mutation, false to use non-atomic batches",
+            ConfigOption.Type.MASKABLE,
+            false);
+
+    // Replication
+    public static final ConfigOption<Integer> REPLICATION_FACTOR = new ConfigOption<>(
+            CQL_NS,
+            "replication-factor",
+            "The number of data replicas (including the original copy) that should be kept. " +
+                    "This is only meaningful for storage backends that natively support data replication.",
```

Comment:
The second sentence doesn't seem to make sense in context of specific backend configuration item